lib/HALTarget/src/Network.cpp

```diff
@@ -95,7 +95,7 @@ bool Network::process()
         /* Nothing to do. */
         isSuccess = true;
         break;
-    
+
     case STATE_SETUP:

```

Comment:
Changes in this file are not related to the PR title.

Reply:
Resolved with [894deb3](https://github.com/BlueAndi/DroidControlShip/pull/30/commits/894deb35cdc54e80f856a7d02bf2ef8635dd5504)